packages/terra-divider/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* Separate div, props, and styling to allow inserting text into divider
+* CSS variables added for theming text and divider
+* Tests added for text divider
```

Comment:
When reading through the changelog it may be easier for consumers if we limit the implementation details and just add an entry for the API changes. 

```md
### Added
* Optional `text` property to display within the center of the divider
* Themeable SCSS variables for the text and divider
```

Reply:
Changes added! [7ad1cc8](https://github.com/cerner/terra-core/pull/2296/commits/7ad1cc8ca8af947a4b40c5cbf02c9a0bf49c534f)